packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -114,17 +127,22 @@ export default class ModalPanel extends React.Component<Props> {
             light,
             onClose,
             style,
+            testId,
         } = this.props;
 
         const mainContent = this.renderMainContent();
 
         return (
-            <View style={[styles.wrapper, !light && styles.dark, style]}>
+            <View
+                style={[styles.wrapper, !light && styles.dark, style]}
+                testId={testId && `${testId}-ModalPanel`}
+            >
                 {closeButtonVisible && (
                     <CloseButton
                         light={!light}
                         onClick={onClose}
                         style={styles.closeButton}
+                        testId={testId && `${testId}-CloseButton`}
```

Comment:
nit: It's a little weird seeing a mix of kebab-case and PascalCase.  Maybe this should be `${testId}-close`.